search/filters/FulltextFilter.php

```diff
@@ -73,5 +73,18 @@ public function getDbName() {
 
 		return parent::getDbName();
 	}
+	
+	private function prepareColumns($columns) {
+		$class = '';
+		$parts = explode(',', $columns);
+		array_walk($parts, function(&$col, $key) use (&$class) {
+			$name = trim($col, " \t\n\r\0\x0B\"");
+			if (empty($class)) {
```

Comment:
`empty` is fine surely? I think the only other option would be `$class === ''` but I don't think that adds any value?